meta-leda-components/recipes-sdv/eclipse-leda/leda-contrib-self-update-agent_git.bb

```diff
@@ -51,6 +51,10 @@ FILES:${PN}-data += " \
     /data/self-update-images/ \
```

Comment:
/data/self-update-images/ can be as in the containerized one /data/selfupdates .Just sow it

Reply:
Fixed, also in the .service file.